### ISSUE TITLE
[2.0.0] Phalcon\Mvc\Model\ValidationFailed wrong parameters for Exception

### ISSUE THREAD

**Issue by hacketo** [phalcon/cphalcon#10208]
This [line](https://github.com/phalcon/cphalcon/blob/master/phalcon/mvc/model.zep#L2756) throw a "Fatal error: Wrong parameters for Exception" as [ValidationFailed](https://github.com/phalcon/cphalcon/blob/master/phalcon/mvc/model/validationfailed.zep) has no such constructor defined

```php
throw new \Phalcon\Mvc\Model\ValidationFailed(this, this->_errorMessages);
```

ref link : [model.zep.c](https://github.com/phalcon/cphalcon/blob/master/ext/phalcon/mvc/model.zep.c#L3415)

**Comment by andresgutierrez** [comment]
This is now fixed in the branch 2.0.x, could you please try again compiling from that branch?

**Comment by hacketo** [comment]
I've compiled from the 2.0.x branch and this is fixed.
